packages/iTwinUI-react/src/core/Table/Table.test.tsx

```diff
@@ -220,28 +220,33 @@ const BooleanFilter = (
   );
 };
 
-const expandAll = (container: HTMLElement, oldExpanders: Element[] = []) => {
+const expandAll = async (
+  container: HTMLElement,
+  oldExpanders: Element[] = [],
+) => {
   const allExpanders = Array.from(
     container.querySelectorAll('.iui-row-expander'),
   );
   const newExpanders = allExpanders.filter((e) => !oldExpanders.includes(e));
-  newExpanders.forEach((button) => userEvent.click(button));
+  for (const button of newExpanders) {
```

Comment:
Was it you or prettier?

Reply:
It was me, because I could not write an async foreach